firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -21,13 +21,14 @@
 void setupConsoleFunctions(){
 
   //particle console function declarations, belongs in setup() as per docs
+  Particle.function("Force_Reset", force_reset); 
+  Particle.function("Turn_Debugging_Publishes_On_Off", toggle_debugging_publishes);   
   Particle.function("Change_Initial_Timer", initial_timer_set); 
   Particle.function("Change_Duration_Timer", duration_timer_set);     
   Particle.function("Change_Stillness_Timer", stillness_timer_set);  
   Particle.function("Change_INS_Threshold", ins_threshold_set);
-  Particle.function("Turn_Debugging_Publishes_On_Off", toggle_debugging_publishes);   
   Particle.function("Change_IM21_Door_ID", im21_door_id_set); 
-
+  
```

Comment:
Oops, looks like you accidentally added some whitespace here. Could you please remove it?

Reply:
For sure